.github/workflows/ci.yml

```
@@ -28,12 +28,15 @@ jobs:
 
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
-
+      -
```

actionlint (1.7.7)

element of "steps" section should not be empty. please remove this section if it's unnecessary

(syntax-check)